src/omero_upload/__init__.py

```diff
@@ -0,0 +1,4 @@
+from library import upload_ln_s
+__all__ = (
```

Comment:
Is this to allow wildcard import? With an empty `__init__.py`,  should ew be able to call `from omero_upload.library import upload_ln_s`?

Reply:
My thinking was to be explicit about the API:
- `omero_upload.upload_ln_s` is public
- `library.py` is an implementation detail